src/layouts/sidebar-sidecar/components/table-of-contents/index.tsx

```diff
@@ -53,7 +64,11 @@ const TableOfContentsListItem = ({ isActive, slug, title }): ReactElement => {
 
 	return (
 		<li className={className} key={slug}>
-			<a className={s.tableOfContentsListItemAnchor} href={`#${slug}`}>
+			<a
+				className={s.tableOfContentsListItemAnchor}
+				href={`#${slug}`}
+				data-heap-track={`toc-list-item-index-${index}`}
```

Comment:
Is there any reason we might want the `slug` here? 💭

Reply:
I had it previously, and then realized it seemed very redundant with `href` 🤔

For example, in Heap I think if we wanted to select an element with a specific slug, we could use the CSS selector `a[data-heap-track*=toc-list-item][href=#<slug>]` for example. But could definitely add it back in if we have a concrete use case for it 👍

Reply:
Ah that makes sense, thanks for the insight!